EIPS/eip-draft_extension_enr_client.md

```diff
@@ -0,0 +1,65 @@
+---
```

Comment:
```suggestion
---
eip: 7636
```
Assigning next sequential EIP/ERC/RIP number.
Numbering changed to sequential from 7500.

Please also update the filename.